### PR TITLE
S2U-13 5.1.1.9 Tests & Quizzes: Improve selector for sub question pools when creating assessments

### PR DESCRIPTION
**Pull request by JuanDavid102**

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/author/editPart.jsp [hunk 1]
```diff
@@ -30,16 +30,51 @@
...
       </head>
       <body onload="<%= request.getAttribute("html.body.onload") %>">
+<script>
+    function clearSelection(selectObject)
+    {
+        for (var i=0; i<selectObject.options.length; i++)
+        {
+            selectObject.options[i].selected=false;
+        }
+        changeSelect(selectObject);
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/jsf/author/editPart.jsp:56]: The function is used before it's defined.

File: samigo/samigo-app/src/webapp/jsf/author/editPart.jsp [hunk 2]
```diff
@@ -30,16 +30,51 @@
...
+    }
 
+    $(document).ready(function() {
+        addTagSelector(document.getElementById('modifyPartForm:assignToPool'));
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/jsf/author/editPart.jsp:60]: the function is used before it's defined.

**Comment by mpellicer** [comment]
I see some code repetition, could we extract common JS code to a JS file and reuse it? Where did you get this code from, existing using of select2 in Samigo?

**Review by mpellicer — CHANGES_REQUESTED** [review]
Some comments and concerns around code repetition.

**Comment by mpellicer** [comment]
After you complete the requested changes, would you mind posting a GIF file or screenshots with the new UI? thanks

File: samigo/samigo-app/src/webapp/jsf/author/editPart.jsp (hunk 1 above)
Comment (JuanDavid102) [reply samigo/samigo-app/src/webapp/jsf/author/editPart.jsp:56]: Ok

**Review by mpellicer — APPROVED** [review]

**Pull request merged**